openassessment/xblock/static/js/src/lms/components/WaitingStepContent.jsx

```diff
@@ -3,7 +3,9 @@ import PropTypes from 'prop-types';
 import { Alert } from '@edx/paragon';
 import WaitingStepList from './WaitingStepList';
 
-const WaitingStepContent = ({ waitingStepDetails, refreshData }) => {
+const WaitingStepContent = ({
+  waitingStepDetails, refreshData, findStudent, selectableLearnersEnabled,
```

Comment:
We should be consistent about our naming conventions. If we're using students then we should use student throughout the implementation:
```suggestion
  waitingStepDetails, refreshData, findStudent, selectableStudentsEnabled,
```

Reply:
Thus, we should use `student` instead of learners throughout the implementation. In this repo, do they call them learners or students?

Reply:
I'll call them student